src/Api/referrals.js

```diff
@@ -47,7 +47,7 @@ export function decodeReferralCode(hexCode) {
 }
 
 export function encodeReferralCode(code) {
-  let final = code.replace(/[^\w\s_]/g, ""); // replace everything other than numbers, string  and underscor to ''
+  const final = code.replace(/[^\w\s_]/g, ""); // replace everything other than numbers, string  and underscor to ''
```

Comment:
@xvi10 that regexp won't replace any space-characters as well, is it on purpose?